### ISSUE TITLE
waybar random crash

### ISSUE THREAD

**Issue by alexminder** [Alexays/Waybar#481]
Waybar v0.8 constantly chashing.

abrt log attached

[waybar.crash.1.dump.txt](https://github.com/Alexays/Waybar/files/3696531/waybar.crash.1.dump.txt)

**Comment by Alexays** [comment]
Can you run gdb and get the backtrace?

**Comment by alexminder** [comment]
```
Core was generated by `waybar -b bar-0'.
Program terminated with signal SIGABRT, Aborted.
#0  0x00006badc17515fb in raise () from /lib64/libc.so.6
[Current thread is 1 (LWP 7582)]
(gdb) bt full
#0  0x00006badc17515fb in raise () at /lib64/libc.so.6
#1  0x00006badc173a539 in abort () at /lib64/libc.so.6
#2  0x00006badc173a411 in  () at /lib64/libc.so.6
#3  0x00006badc1748d32 in  () at /lib64/libc.so.6
#4  0x00006badc1c8038e in mpd_error_get_message (error=<optimized out>) at ../libmpdclient-2.16/src/ierror.h:130
        __PRETTY_FUNCTION__ = "mpd_error_get_message"
#5  0x00006badc1c8038e in mpd_connection_get_error_message (connection=<optimized out>) at ../libmpdclient-2.16/src/error.c:43
#6  0x0000040bd58b221c in waybar::modules::MPD::checkErrors(mpd_connection*) (this=0x40c12152000, conn=0x40c11f1cec0) at ../Waybar-0.8.0/src/modules/mpd.cpp:278
        error_message = <optimized out>
#7  0x0000040bd58b2376 in waybar::modules::MPD::waitForEvent() (this=0x40c12152000) at ../Waybar-0.8.0/src/modules/mpd.cpp:309
        conn = 0x40c11f1cec0
#8  0x0000040bd58b4417 in waybar::modules::MPD::<lambda()>::operator() (__closure=<optimized out>) at ../Waybar-0.8.0/src/modules/mpd.cpp:66
#9  0x0000040bd58b4417 in std::__invoke_impl<void, waybar::modules::MPD::event_listener()::<lambda()> > (__f=...) at /usr/lib/gcc/x86_64-pc-linux-gnu/8.3.0/include/g++-v8/bits/invoke.h:60
#10 0x0000040bd58b4417 in std::__invoke<waybar::modules::MPD::event_listener()::<lambda()> > (__fn=...) at /usr/lib/gcc/x86_64-pc-linux-gnu/8.3.0/include/g++-v8/bits/invoke.h:95
#11 0x0000040bd58b4417 in std::thread::_Invoker<std::tuple<waybar::modules::MPD::event_listener()::<lambda()> > >::_M_invoke<0> (this=<optimized out>)
    at /usr/lib/gcc/x86_64-pc-linux-gnu/8.3.0/include/g++-v8/thread:244
#12 0x0000040bd58b4417 in std::thread::_Invoker<std::tuple<waybar::modules::MPD::event_listener()::<lambda()> > >::operator() (this=<optimized out>)
    at /usr/lib/gcc/x86_64-pc-linux-gnu/8.3.0/include/g++-v8/thread:253
#13 0x0000040bd58b4417 in std::thread::_State_impl<std::thread::_Invoker<std::tuple<waybar::modules::MPD::event_listener()::<lambda()> > > >::_M_run(void) (this=0x40c1215d910)
    at /usr/lib/gcc/x86_64-pc-linux-gnu/8.3.0/include/g++-v8/thread:196
#14 0x00006badc1b4d18e in std::execute_native_thread_routine(void*) (__p=0x40c1215d910) at /var/tmp/portage/sys-devel/gcc-8.3.0-r1/work/gcc-8.3.0/libstdc++-v3/src/c++11/thread.cc:80
        __t = 
                  {_M_t = {_M_t = {<std::_Tuple_impl<0, std::thread::_State*, std::default_delete<std::thread::_State> >> = {<std::_Tuple_impl<1, std::default_delete<std::thread::_State> >> = {<std::_Head_base<1, std::default_delete<std::thread::_State>, true>> = {<std::default_delete<std::thread::_State>> = {<No data fields>}, <No data fields>}, <No data fields>}, <std::_Head_base<0, std::thread::_State*, false>> = {_M_head_impl = 0x40c1215d910}, <No data fields>}, <No data fields>}}}
#15 0x00006badc18f2408 in  () at /lib64/libpthread.so.0
#16 0x00006badc182052f in clone () at /lib64/libc.so.6
(gdb)
```

**Comment by Alexays** [comment]
Already fixed with e6599d8ed5f3391d7f060678c2b662263a4ecd2b
